### PR TITLE
Allow adding HTML landing pages to spaces

### PR DESCRIPTION
**Pull request by craxrev**
These changes add an HTML option when creating new landing page, this will allow space admins to add their custom HTML landing pages. Still no sanitization is made yet, just testing it out.

### CODE REVIEW

File: controllers/ContainerController.php [hunk 1]
```diff
@@ -70,7 +70,9 @@ public function actionView()
         } elseif ($page->type == Container::TYPE_MARKDOWN) {
             return $this->render('markdown', array('page' => $page, 'md' => $page->page_content));
         } elseif ($page->type == Container::TYPE_TEMPLATE) {
-            return $this->viewTemplatePage($page);
+	        return $this->viewTemplatePage($page);
+        } elseif ($page->type == Container::TYPE_HTML) {
+	        return $this->render('html', ['page' => $page, 'html' => $page->page_content, 'title' => $page->title]);
         } elseif ($page->type == Container::TYPE_PHP) {
             return $this->render('php', [
```
Comment (bumi) [inline controllers/ContainerController.php:77]: the indentation is a bit off in the return statements (compared to e.g. line 71)

File: controllers/ContainerController.php [hunk 2]
```diff
@@ -70,7 +70,9 @@ public function actionView()
         } elseif ($page->type == Container::TYPE_MARKDOWN) {
             return $this->render('markdown', array('page' => $page, 'md' => $page->page_content));
         } elseif ($page->type == Container::TYPE_TEMPLATE) {
-            return $this->viewTemplatePage($page);
+	        return $this->viewTemplatePage($page);
+        } elseif ($page->type == Container::TYPE_HTML) {
+	        return $this->render('html', ['page' => $page, 'html' => $page->page_content, 'title' => $page->title]);
```
Comment (bumi) [inline controllers/ContainerController.php]: `$page->page_content` and `$page->title` page title is what we need to sanitize?

**Review by bumi — CHANGES_REQUESTED** [review]
looks good for me! let's deploy it an test it. 

I'd suggest to fix the indentation (see my comment)

File: controllers/ContainerController.php (hunk 2 above)
Comment (craxrev) [reply controllers/ContainerController.php]: yep, mainly `$page->page_content`

File: controllers/ContainerController.php (hunk 1 above)
Comment (craxrev) [reply controllers/ContainerController.php:77]: fixed

**Review by bumi — APPROVED** [review]

**Pull request merged**